site/faq.md

```diff
@@ -23,7 +23,7 @@ OSS Scope has no user concept, this is only available in Weave Cloud. To limit t
 
 - setup a reverse proxy with auth and block access to non admin users,
 - capture the calls with something like Chrome network console to get the endpoints to know which requests to authenticate in the proxy server.
-- you can use Basic HTTP Auth since Scope 1.10.0 - just use these command line
+- you can use Basic HTTP Auth since Scope 1.10.0 (support for loading password from file sice Scope 1.13.3) - just use these command line
```

Comment:
Typo s/sice/since/

Put this in a separate PR so we don't publish incorrect info for a time.

Reply:
fixed, faq moved to #3852